src/calendar/header/index.tsx

```diff
@@ -90,6 +92,7 @@ const CalendarHeader = forwardRef((props: CalendarHeaderProps, ref) => {
     renderArrow,
     onPressArrowLeft,
     onPressArrowRight,
+    arrowsHitSlop,
```

Comment:
add the default here as well, i.e. `arrowsHitSlop = 20`

Reply:
Added